website/docs/d/is_volume.html.markdown

```diff
@@ -43,6 +43,7 @@ Review the argument references that you can specify for your data source.
 ## Attribute reference
 In addition to all argument reference list, you can access the following attribute references after your data source is created.
 
+- `access_tags`  - (String) Access management tags associated for the volume.
```

Comment:
`access_tags` - (Optional, List of Strings)

Reply:
Changed to (List of Strings) as it is a computed field